plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -287,11 +290,13 @@ private function get_feature_definitions() {
 				),
 				'remote_logging'         => array(
 					'name'               => __( 'Remote Logging', 'woocommerce' ),
-					'description'        => __(
-						'Enable this feature to log errors and related data to Automattic servers for debugging purposes and to improve WooCommerce',
-						'woocommerce'
+					'description'        => sprintf(
+						/* translators: %1$s: opening link tag, %2$s: closing link tag */
+						__( 'Allow WooCommerce to send error logs and non-sensitive diagnostic data to help improve WooCommerce. This feature requires %1$susage tracking%2$s to be enabled.', 'woocommerce' ),
+						'<a href="' . admin_url( 'admin.php?page=wc-settings&tab=advanced&section=woocommerce_com' ) . '">',
+						'</a>'
 					),
-					'enabled_by_default' => true,
+					'enabled_by_default' => WC_Site_Tracking::is_tracking_enabled(),
```

Comment:
Can you use the `$tracking_enabled` value instead?

Reply:
Sure! Updated it 438cbc766f9beb0ecf6647269c218d94c346284f.